tests/NotifierTest.php

```diff
@@ -197,4 +197,31 @@ public function testKeysBlacklist()
             'key3' => ['key1' => '[Filtered]'],
         ], $notifier->notice['params']);
     }
+
+    public function testNestedException()
+    {
+        $notifier = $this->newNotifier();
+        try {
+            throw new \Exception('foo');
```

Comment:
Use Troublemaker to generate exception so the source line does not change as this test is changed